modules/combined/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -270,7 +270,7 @@
   solve_type = 'PJFNK'
 
   petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
+  petsc_options_value = 'lu            superlu_dist'
```

Comment:
Minor but why all the extra whitespace in each of these input files. Usually we line the PETSc values up with their corresponding options. It doesn't appear it was like that in these particular files before, but it's not after this change either.